src/index.js

```diff
@@ -12,13 +12,13 @@ instanceOfMainClass.collectAllLinks();
 // initialization of btnClear for Category
 document.querySelector("#btnClear").addEventListener("click", (e) => {
 e.preventDefault();
-btnClearCategoriesFilter();
+btnClearForCategories();
 });
 
 // initialization of btnClear for Links
 document.querySelector("#btnClearLinks").addEventListener("click", (e) => {
 e.preventDefault();
-btnClearLinksFilter();
+btnClearForLinks();
```

Comment:
bad naming, do clear naming, when we have clear text filter, or select.
also
why you did it directly without function as I required yesterday. Is better if is in function implemented dom manipulation then directly, and call function in index, but implement it in ui file.
btnClearForCategories
btnClearForLinks
not clear is for text/select